### PR TITLE
Hshapiro/remove signal r

### PR DESCRIPTION
**Pull request by heatherbshapiro**
Removed the signalR logo by hard coding userData.cs #31

### CODE REVIEW

File: src/JabbR-Core/Hubs/Chat.cs [hunk 1]
```diff
@@ -1,17 +1,98 @@
...
+                new { Name = "Alt + Number", Group = "shortcut", IsKeyCombination = true, Description = LanguageResources.Client_ShortcutSpecificTab }
+            };
+        }
+
+        public void LoadRooms()
+        {
+
+        }
+
+        public void updateActivity()
```
Comment (davidfowl) [inline src/JabbR-Core/Hubs/Chat.cs]: C# = Pascal casing for method names. First letter uppercase.

File: src/JabbR-Core/Hubs/Chat.cs [hunk 2]
```diff
@@ -1,17 +1,98 @@
...
+                FlagClass = null,
+                Flag = null,
+                Country = null,
+                LastActive = "2016-08-23 00:26:35.713",
+                TimeAgo = null,
+                Admin = true,
+                Afk = true,
+            };
+
+            var hack = new List<LobbyRoomViewModel>();
```
Comment (davidfowl) [inline src/JabbR-Core/Hubs/Chat.cs]: it's a hack but call it rooms? 😄

File: src/JabbR-Core/Hubs/Chat.cs [hunk 3]
```diff
@@ -1,17 +1,98 @@
...
+﻿using System.Collections.Generic;
 using Microsoft.AspNetCore.SignalR;
+using JabbR_Core.ViewModels;
 
 namespace JabbR_Core.Hubs
 {
+
     public class Chat : Hub
     {
+        public List<string> Rooms;
```
Comment (davidfowl) [inline src/JabbR-Core/Hubs/Chat.cs]: What does this represent and why is it always null?

File: src/JabbR-Core/Hubs/Chat.cs (hunk 1 above)
Comment (KSLHacks) [inline src/JabbR-Core/Hubs/Chat.cs]: Fixed.

File: src/JabbR-Core/Hubs/Chat.cs (hunk 2 above)
Comment (KSLHacks) [inline src/JabbR-Core/Hubs/Chat.cs]: Okay.. :crying_cat_face:

File: src/JabbR-Core/Hubs/Chat.cs (hunk 3 above)
Comment (KSLHacks) [inline src/JabbR-Core/Hubs/Chat.cs]: Used as a placeholder for `UpdateActivity(UserData user, List<string> rooms)`. Works without the parameter but wanted to keep signature the same.

File: src/JabbR-Core/Hubs/Chat.cs [hunk 4]
```diff
@@ -1,17 +1,98 @@
...
             Clients.Caller.logOn(new object[0], new object[0], new { TabOrder = new List<string>() });
         }
+
+        public List<LobbyRoomViewModel> GetRooms()
+        {
+            //List<string> rooms;
+            UserData user = new UserData()
+            {
+                Name = "light_meow",
+                Hash = null,
```
Comment (adamtuliper) [inline src/JabbR-Core/Hubs/Chat.cs]: Get rid of nulls here, also use var. Example var user = new UserData()...

File: src/JabbR-Core/Hubs/Chat.cs [hunk 5]
```diff
@@ -1,17 +1,98 @@
...
+                Afk = true,
+            };
+
+            var rooms = new List<LobbyRoomViewModel>();
+            
+            rooms.Add(new LobbyRoomViewModel
+            {
+                Name = user.Name,
+                Count = 1,
+                Private = false,
```
Comment (adamtuliper) [inline src/JabbR-Core/Hubs/Chat.cs]: false are defaults, could avoidusing here too

File: src/JabbR-Core/wwwroot/Chat.js [hunk 6]
```diff
@@ -202,7 +202,7 @@
     }
 
     function getUserViewModel(user, isOwner) {
-        var lastActive = user.LastActivity.fromJsonDate();
```
Comment (adamtuliper) [inline src/JabbR-Core/wwwroot/Chat.js]: Default in the model coming to the client (ie server side)

File: src/JabbR-Core/userData.cs [hunk 7]
```diff
@@ -0,0 +1,26 @@
+﻿namespace JabbR_Core
+{
+    // Class object to replace repo and EF models.. hard coded in Chat.cs for invoke functions
+    // user data is gathered upon login/auth which populates models
+
+    public class UserData
```
Comment (davidfowl) [inline src/JabbR-Core/userData.cs]: Why fields and not properties ?

Comment (adamtuliper) [inline src/JabbR-Core/userData.cs]: Heather, just add { get; set; } to each at the end of each line ex public string Name { get; set; }

Comment (KSLHacks) [inline src/JabbR-Core/userData.cs]: Will Fix and submit new PR.

**Pull request merged**